scipy/stats/morestats.py

```diff
@@ -3478,3 +3478,74 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         R = np.minimum(1, hypot(sin_mean, cos_mean))
 
     return ((high - low)/2.0/pi) * sqrt(-2*log(R))
+
+
+def quartile_coeff_dispersion(samples, q=(0.25, 0.75), axis=None, interpolation='lower'):
+    """
+    Compute the quartile coefficient of dispersion for a given sample array.
+
+    Given two quartiles Q1, Q2 from array samples such that Q1 < Q2, the quartile
+    coefficient of dispersion is calculated by (Q2 - Q1)/(Q1 + Q2).
+    by default it is computed using 3rd and 1st (Q2, Q1 respectively) quartiles.
```

Comment:
```suggestion
    By default it is computed using 3rd and 1st (Q2, Q1 respectively) quartiles.
```
Consider whether the order here (Q2, Q1) should match the order above; e,g, "By default, it is computed using the 1st and 3rd..."
Also, Q1, Q2, and Q3 are common abbreviations for 25th percentile, median, and 75th percentile, so it may be confusing to refer to the 75th percentile as Q2 here.

Reply:
Good point.
Maybe using `Q_high`, `Q_low` would be better.